client/package.json

```diff
@@ -13,10 +13,10 @@
     "@testing-library/react": "^13.2.0",
     "@testing-library/user-event": "^13.5.0",
     "axios": "^0.27.2",
-    "react": "^18.1.0",
+    "react": "^18.3.1",
```

Comment:
@Jaycee-9 Best to keep versions intact unless needed for a feature implementation or bug fix.

Reply:
@roshnet  sure I'll change it to 18.1.0